config/prometheus/monitoring-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
I think I'd prefer a single file containing all of the prometheus config (just separate resources with:
```
---
```
See `local-argo-deployment.yml` as an example.

The reason being that I'd always want to deploy all of them or none of them. It also makes it easy for me to delete the resources with one command (`kubectl delete -f ./config/prometheus-resources.yaml`)